client/components/club-details/EventsHostedByClub.tsx

```diff
@@ -0,0 +1,119 @@
+import styled from 'styled-components'
+import moment from 'moment-timezone'
+import { Category } from '../Category'
+import { colours, desktopFontSize, device, fontWeight } from '../../styles'
+import { BasicEvent } from '../../context/Base'
+
+const mobile = `425px`
+
+const largerThan = (size: string): string => `(min-width: ${size})`
+
+const EventsTitle = styled.h2`
+  margin-top: 40px;
+  margin-bottom: 32px;
+`
+
+const EventCardDetails = styled.div`
+  margin-top: 0;
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-bottom: 20px;
+
+  width: 100%;
+`
+
+const EventCardTitle = styled.p`
+  font-weight: ${fontWeight.bold};
+  margin-bottom: 0;
+`
+
+// TODO: make box shadow cleaner
+const EventCardWrapper = styled.div`
+  box-shadow: 0px 4px 4px 1px ${colours.neutralLight1};
+  display: flex;
+  margin-top: 10px;
+`
+
+const EventCardImg = styled.img`
+  width: 80px;
+
+  @media ${largerThan(mobile)} {
+    width: 180px;
+  }
+`
+
+const EventCardDate = styled.p`
+  margin-top: 5px;
+`
+
+const PastEventsTitle = styled.p`
+  font-size: ${desktopFontSize.subtitle1};
+  margin-top: 40px;
+  margin-bottom: 24px;
+`
+
+export const EventsHostedByClub = ({ events }: { events: Array<BasicEvent> }) => {
+  console.log(events)
+  const upcomingEvents = events.filter((e) => e.startTime.isAfter(moment()))
+
+  const pastEvents = events.filter((e) => e.startTime.isBefore(moment()))
+
+  return (
+    <div>
+      <EventsTitle>Upcoming Events</EventsTitle>
+      {upcomingEvents.map((e, i) => {
+        return <EventCard event={e} key={`event-card-${i}`} />
+      })}
+      <PastEventsTitle>Past Events</PastEventsTitle>
+      {pastEvents.map((e, i) => {
+        return <EventCard event={e} key={`past-event-card-${i}`} />
+      })}
+    </div>
+  )
+}
+
+export const EventCard = ({ event }: { event: BasicEvent }) => {
+  const date = `${event.startTime.local().format('dddd')}, ${event.startTime
+    .local()
+    .format('LL z')}`
+  const time = `${event.startTime.format('ha z')} - ${event.endTime.format('ha z')}`
```

Comment:
like justin said, it would be nice documentation wise to add a comment to show the format. In my PR, I added `//Format: 8:55 PM - 9:55 PM EST` as a comment.

Reply:
added